disco_aws_automation/exceptions.py

```diff
@@ -22,6 +22,11 @@ class EarlyExitException(Exception):
     pass
 
 
+class ProgrammerError(Exception):
+    "An exception state that resulted from a coding error, not an environment error."
```

Comment:
The intention is that it would be—it's only used in one place, and obviously I can't control how other people use it, so... 😁 

But the idea is "this is not because you're not supplying the right config file, this is because you wrote the wrong code."  So, that's the justification for that docstring.